docker/models/containers.py

```diff
@@ -285,7 +285,8 @@ def logs(self, **kwargs):
                 datetime or integer epoch (in seconds)
 
         Returns:
-            (generator or str): Logs from the container.
+            (generatoran iterator of output bytes or bytes):
```

Comment:
Same as a above. And you had a little typo here `generatoran` -> `generator`

Reply:
Thanks for your review and comment, I will change it.